imports/plugins/core/checkout/client/templates/cartIcon/cartIcon.js

```diff
@@ -1,15 +1,16 @@
 import { $ } from "meteor/jquery";
```

Comment:
Not directly related.. but what is jquery used for here!?!

Reply:
Many of the Blaze templates still use it to select elements. We should be able to remove it once we remove all Blaze.